core/block_svg.js

```diff
@@ -644,18 +633,17 @@ BlockSvg.prototype.getBoundingRectangle = function() {
     left = blockXY.x;
     right = blockXY.x + blockBounds.width;
   }
-  return new Rect(
-      blockXY.y, blockXY.y + blockBounds.height, left, right);
+  return new Rect(blockXY.y, blockXY.y + blockBounds.height, left, right);
 };
 
 /**
  * Notify every input on this block to mark its fields as dirty.
  * A dirty field is a field that needs to be re-rendered.
  */
 BlockSvg.prototype.markDirty = function() {
-  this.pathObject.constants =
-    (/** @type {!WorkspaceSvg} */ (this.workspace))
-        .getRenderer().getConstants();
+  this.pathObject.constants = (/** @type {!WorkspaceSvg} */ (this.workspace))
+                                  .getRenderer()
+                                  .getConstants();
```

Comment:
Could you change the indentation here to be +4 spaces? [Horizontal alignment is discouraged](https://engdoc.corp.google.com/eng/doc/devguide/js/styleguide.md?cl=head#formatting-horizontal-alignment) so idk why clang does this.

Reply:
Done